FusionLocalAuth/Sources/FusionLocalAuth_Common/LocalAuthManager.swift

```diff
@@ -97,7 +139,8 @@ public protocol LocalAuthManagerProtocol {
      *        cancelText: String
      *
      */
-    func requestBiometricAuthentication(authTitle:String, reasonTitle:String, cancelText:String, status: @escaping (AuthError?) -> Void )
+    
+    func requestBiometricAuthentication(authTitle:String, reasonTitle:String, cancelTitle:String, completionStatus: @escaping (Bool, AuthError?) -> Void )
```

Comment:
Just remembered, I used the "on<Something>" notion for callbacks in my code before, like "onComplete", "onError", "onSuccess" etc., it's not a must, but if you like, you can use it also.

Reply:
Sure will use it. Thanks for letting me know.